src/client/automation/playback/press/key-press-simulator.js

```diff
@@ -73,10 +72,10 @@ export default class KeyPressSimulator {
     }
 
     _addKeyPropertyToEventOptions (eventOptions) {
-        if (keyIdentifierRequiredForEvent())
+        if (browserUtils.isSafari)
```

Comment:
I guess we should use the feature detection in this case:
```js
if ('key' in KeyboardEvent.prototype)
   eventOptions.key = ...

if ('keyIdentifier' in KeyboardEvent.prototype)
   eventOptions.keyIdentifier = ...
```

Reply:
ok